### PR TITLE
Enhancements to Stake page

### PR DESCRIPTION
**Pull request by skeeter6380**
I added the price to the top of the Stake page (fixed resulting layout issues for mobile); I also added a listing for a wallet's wrapped staked sb token; added a new section below staked grid to use the price and token totals to calculate balances.

### CODE REVIEW

**Comment by skeeter6380** [comment]
Oops, just realized I forgot to calculate the value of the wrapped tokens correctly using the Index... i'll fix that tomorrow.

**Comment by skeeter6380** [comment]
ok, calculation is fixed

File: src/store/slices/account-slice.ts [hunk 1]
```diff
@@ -77,8 +83,15 @@ export const loadAccountDetails = createAsyncThunk("account/loadAccountDetails",
         unstakeAllowance = await ssbContract.allowance(address, addresses.STAKING_ADDRESS);
     }
 
+    if (addresses.WSSB_ADDRESS) {
+        const wssbContract = new ethers.Contract(addresses.WSSB_ADDRESS, MemoTokenContract, provider);
+        wssbBalance = await wssbContract.balanceOf(address);
+        // unstakeAllowance = await wssbContract.allowance(address, addresses.STAKING_ADDRESS);
```
Comment (0xOtoguro) [inline src/store/slices/account-slice.ts]: Shouldn't we remove this commented assignation?

Comment (0xOtoguro) [reply src/store/slices/account-slice.ts]: if you want to keep it for later usage, in that case could you please add a comment below that commented line to explain why it's there

Comment (skeeter6380) [reply src/store/slices/account-slice.ts]: oh, i actually didn't know what that did.  I had copied that section from the previous ones and modified, accordingly.  I'll just remove it

**Review by 0xOtoguro — APPROVED** [review]

**Pull request merged**